test-data/unit/check-basic.test

```diff
@@ -372,3 +372,21 @@ none = None
 b = none.__bool__()
 reveal_type(b)  # E: Revealed type is 'builtins.bool'
 [builtins fixtures/bool.pyi]
+
+[case testAssignmentInvariantNoteForList]
+from typing import List
+x: List[int]
+y: List[float]
+y = x # E: Incompatible types in assignment (expression has type "List[int]", variable has type "List[float]") \
+     # N: "List" is invariant -- see http://mypy.readthedocs.io/en/latest/common_issues.html#variance \
+     # N: Consider using "Sequence" instead, which is covariant
+[builtins fixtures/list.pyi]
+
+[case testAssignmentInvariantNoteForDict]
+from typing import Dict
+x: Dict[str, int]
+y: Dict[str, float]
+y = x # E: Incompatible types in assignment (expression has type "Dict[str, int]", variable has type "Dict[str, float]") \
+     # N: "Dict" is invariant -- see http://mypy.readthedocs.io/en/latest/common_issues.html#variance \
+     # N: Consider using "Sequence" instead, which is covariant
```

Comment:
Here it should be `"Mapping"`, not `"Sequence"`.

Reply:
Yeah! It's right.